pepper-apis/src/test/java/org/broadinstitute/ddp/route/PutFormAnswersRouteStandaloneTest.java

```diff
@@ -91,8 +91,7 @@
 import org.junit.Test;
 import spark.HaltException;
 
-
-public class PutFormAnswersRouteStandaloneTest {
+public class PutFormAnswersRouteStandaloneTest extends IntegrationTestSuite.TestCaseWithCacheEnabled {
```

Comment:
If I'm understanding correctly, I think `TestCaseWithCacheEnabled` is the right thing to use.